src/_posts/2016-08-26-Continuous-Integration-on-iOS-with-HockeyApp-and-Testflight-Deployment.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Continuous Integration With Deployment to TestFlight and HockeyApp
 date: 2016-08-26 10:00:00
-tags: ios,ci,continuous integration,fastlane,
+tags: [ios,ci,continuous integration,fastlane]
```

Comment:
If time, it could be nice to find an image for all posts

Reply:
I have corrected the tag format in all posts.
That will be nice but also we can have 2-3 default images for each categories and we can show them for posts without an image by randomly.